src/components/TextArea/TextArea.tsx

```diff
@@ -1,11 +1,9 @@
 import React, { forwardRef } from 'react';
 import { bem } from '../../utils';
 import styles from './TextArea.scss';
-import { Context, Size } from '../../constants';
+import { Size } from '../../constants';
 
 export interface Props extends React.HTMLAttributes<HTMLElement> {
-    /** The textarea context (e.g. brand, primary, bad, good etc. - defaults to brand) */
-    context?: Context;
```

Comment:
I'm not sure we can get away without context here. I'm thinking of a form that has some validation that has a textArea in it, and validation fails for some reason.